### ISSUE TITLE
switch to go-msi for msi backend.

### ISSUE THREAD

**Issue by xoviat** [xoviat/subzero#56]
the example [wix.json](https://github.com/mh-cbon/go-msi/blob/master/wix.json) is considerably easier to generate than the xml files. Also, this removes the lxml requirement.

**Comment by xoviat** [comment]
For reference:


```
{
  "product": "go-msi",
  "company": "mh-cbon",
  "license": "LICENSE",
  "upgrade-code": "8615055C-D8E0-404C-93BE-441C503BA6F0",
  "files": {
    "guid": "378896D8-6749-4821-870A-44CBBB791D0C",
    "items": [
      "go-msi.exe"
    ]
  },
  "directories": [
    "templates"
  ],
  "env": {
    "guid": "0CB88C7F-85A7-4986-B6CE-1CAD5C17EA0E",
    "vars": [
      {
        "name": "PATH",
        "value": "[INSTALLDIR]",
        "permanent": "no",
        "system": "no",
        "action": "set",
        "part": "last"
      }
    ]
  },
  "shortcuts": {
    "guid": "6DAFD205-3D2D-43D7-BF78-33BFE8746D2A",
    "items": [
      {
        "name": "go-msi",
        "description": "Easy msi pakage for Go",
        "target": "[INSTALLDIR]\\go-msi.exe",
        "wdir": "INSTALLDIR",
        "arguments": ""
      }
    ]
  },
  "choco": {
    "description": "Easy way to generate msi package for a Go project",
    "project-url": "https://github.com/mh-cbon/go-msi",
    "tags": "generate go msi nuget",
    "license-url": "https://github.com/mh-cbon/go-msi/blob/master/LICENSE"
  }
}
```

**Comment by xoviat** [comment]
closed in 3fb011dba7f1c593bfe5d3843763e6c51b21789e.
